### PR TITLE
Added Notifications component under Organisms route

### LINKED ISSUE #867: Notification cards and the page UI
**Describe the solution you'd like**
Implementation of the attached Notifications UI based on [code guidlines](https://github.com/Ignitus/Ignitus-client/wiki/CodeGuidelines-And-Conventions).

This component needs to be under [Organisms](https://github.com/Ignitus/Ignitus-client/tree/master/src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms) 

**Additional context**
Refer [Figma](https://www.figma.com/file/BrP5RowC7va50HyETZQUDA/Ignitus-Design?node-id=121%3A8) for User-Interface and other styles-related issue.

- You can view all the CSS, colors, typography, shadow effects, etc required to implement the design in the [ignitus-DesignSystem](https://github.com/Ignitus/Ignitus-client/tree/master/src/ignitus-Shared/ignitus-DesignSystem) folder. Please try to follow the same pattern. 
- You can download SVG icons from the figma and add them in [ignitus-Assets](https://github.com/Ignitus/Ignitus-client/tree/master/src/ignitus-Shared/ignitus-DesignSystem/ignitus-Assets) folder accordingly.

Screenshot 2020-06-10 at 10 44 59 AM

### PR DESCRIPTION
**Pull request by Dishebh**
Fixes #867 

Preview Link: https://deploy-preview-898--peaceful-nightingale-55a1b7.netlify.app/interface/notifications

### CODE REVIEW

File: src/ignitus-UserInterfaceBook/Components/Organisms/interfaceNotifications/style.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  flex: 1;
+`;
+
+export const StyledNotifications = styled(Notifications)`
+  height: 2rem;
+  width: 70%;
+  padding: 1rem;
+`;
+
+export const Text = styled.span`
```
Comment (Paarmita) [inline src/ignitus-UserInterfaceBook/Components/Organisms/interfaceNotifications/style.ts:19]: For the normal text use the paragraph tag from the styling and for bold choose from the defined typography.

**Comment by Dishebh** [comment]
Preview link: https://deploy-preview-898--peaceful-nightingale-55a1b7.netlify.app/interface/notifications

**Comment by Dishebh** [comment]
Great. Ready for merge @divyanshu-rawat :+1:

**Comment by divyanshu-rawat** [comment]
@Dishebh will review it by evening, will also open some new issues great work :)

File: src/ignitus-UserInterfaceBook/Components/Organisms/interfaceNotifications/style.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+export const StyledIcon = styled(Icon)`
+  width: 1.4rem;
+  fill: ${props => props.color};
+  transition: 0.3s;
+  flex: 1;
+  margin: 0.03rem 0.03rem 0.03rem 10rem;
+`;
+
+export const StyledInterface = styled(Interface)`
```
Comment (divyanshu-rawat) [inline src/ignitus-UserInterfaceBook/Components/Organisms/interfaceNotifications/style.ts]: Not needed Interface does the job well.

**Pull request merged**